tests/unit/testsuite/Bolt/Boltpay/Model/ShippingAndTaxTest.php

```diff
@@ -73,7 +73,8 @@ public function testApplyShippingAddressToQuote() {
             'region' => 'California'
         );
         $result = $this->currentMock->applyShippingAddressToQuote($quote, $shippingAddress);
-
+        array_walk( $result, function(&$value, $key){ $value = trim($value); } );
```

Comment:
remove this

Reply:
@daisy1754, perhaps it maybe good to add this to the main plugin code.  Trailing newlines and spaces have no place in this context and in fact can cause problems with Magento's algorithms